src/corner_kick/src/components/LayersPanel/index.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * This file defines the UI to control the various layers of the visualizer
```

Comment:
Are we referring to the layers that we draw over the "field" section of the visualizer here? Or over the entire visualizer?

Reply:
We referring to the UI that controls layer order and visibility. It will go in the sidebar.

Reply:
Oh lol, referring to every layer of the visualizer, field is just one of them

Reply:
Ah, could we make that a bit more clear here? My confusion was coming from the fact that we're using the word "layer" to refer to the layers of shapes that we draw over the field. Would be good to indicate that this is not the same thing.

Reply:
Also to clarify, the "Layers" here still appear on the "Field Area" of the Visualizer right? We can't draw random shapes that appear magically in other places, right?

Reply:
Okay... Hum I'm not sure what you are talking about. The UI described in this PR is meant to control the z-ordering and visibility of the layers in the visualizer. I suppose by "field area" you are talking about the area where the content (ie. shapes) of the layers are drawn. That "field area" needs additional information (ie. in which order should layers be drawn; is a layer visible or not) which the user can control through the UI in this PR

Reply:
Sorry, I think "Canvas" is a better term. The area where the AI can request shapes to be drawn is separate from the console on the bottom, and control panel on the left?

Reply:
I suppose you could say that. More specifically however is the AI can send shapes to be drawn to the visualizer, and the visualizer can choose where to draw those shapes and add information it deems necessary for visualization. This information can come from the user via additional controls in the Sidebar or Console area (in fact — this is what this PR provides, a control to control the visibility and z-ordering of layers, which is something that the AI does not provide).

TLDR: we need planning documents.

Reply:
We're good

Reply:
This makes sense, could we rephrase the comment to make the above more clear? Perhaps something like: "This file defines the UI to control the layers in the field area of the visualizer"?

Reply:
"field area" isn't the correct term to use here (and was the cause of the confusion above). If anything, I think all we need here is to specify the layers are on the "canvas".

Reply:
Update file description to "This file defines the UI to control the layers in the Canvas"